frontends/web/src/components/language/language.tsx

```diff
@@ -66,17 +65,9 @@ const defaultLanguages = [
 ] as TLanguagesList;
 
 class LanguageSwitch extends Component<Props, State> {
-    constructor(props) {
+    constructor(props: Props) {
         super(props);
         const languages = this.props.languages || defaultLanguages;
-        if (extraLanguages) {
-            extraLanguages.split(',').forEach(code => {
-                languages.push({
-                    code,
-                    display: code,
-                } as TLanguage);
-            });
-        }
```

Comment:
Why did we remove this again?

was extraLanguages only used for testing maybe?

Reply:
This was removed on 29a0764ea4a5d4f412ec928055439a083c47d99d